korrekt/src/io/analyzer_io.rs

```diff
@@ -114,11 +115,11 @@ pub fn retrieve_user_input_for_analyzer_type() -> Result<AnalyzerType> {
     const UNCONSTRAINED_CELLS: i64 = 3;
     const UNDERCONSTRAINED_CIRCUITS: i64 = 4;
 
-    println!("Choose the mode of analysis for your circuit.");
-    println!("1. Unused Gates");
-    println!("2. Unused Columns");
-    println!("3. Unconstrained Cells");
-    println!("4. Underconstrained Circuit");
+    info!("Choose the mode of analysis for your circuit.");
+    info!("1. Unused Gates");
+    info!("2. Unused Columns");
+    info!("3. Unconstrained Cells");
+    info!("4. Underconstrained Circuit");
```

Comment:
Should these actually be `info` over `println`? I am not sure -- but I would have thought options go to stdout by default. Is there a rust convention that should be followed? Not a scientific answer, but https://users.rust-lang.org/t/how-to-get-user-input/5176/11 suggests `println`.
Or was there another reason to implement it this way?

Reply:
Yes, that's correct. For running benchmarks I disabled all the `println`.

However, upon further review, the `retrieve_user_input_for_analyzer_type()` function could be removed, as we now receive user inputs through CLI options.